docs/features/software-templates/migrating-from-v1beta2-to-v1beta3.md

```diff
@@ -0,0 +1,148 @@
+---
+id: migrating-from-v1beta2-to-v1beta3
+title: Migrating to v1beta3 templates
+# prettier-ignore
+description: How to migrate your existing templates to beta3 syntax
+---
+
+# What's new?
```

Comment:
Fine work! ❤️  Does existing scaffolder docs need updating as well?

Reply:
They do, but I was gonna get a few select people to test it out first before blessing this as golden path.